inc/zoo/swar/SWAR.h

```diff
@@ -263,11 +260,9 @@ template<int NBits, typename T>
 struct BooleanSWAR: SWAR<NBits, T> {
     using Base = SWAR<NBits, T>;
 
-    template <std::size_t N, typename = std::enable_if_t<N == Base::Lanes, T>>
-    constexpr BooleanSWAR(Literals_t<NBits, T>, const bool (&values)[N]) : Base{0} {
-        constexpr auto msbOfFirstLane = T{1} << (NBits - 1);
-        this->m_v = Base::loadIntoLanes(values, [](auto x) { return x ? msbOfFirstLane : 0; });
-    }
+    template <std::size_t N>
+    constexpr BooleanSWAR(Literals_t<NBits, T>, const bool (&values)[N])
+       : Base(Literals<NBits, T>, values) { this->m_v << (NBits - 1); }
```

Comment:
My typo in the comment, this has no effect, change to `<<=`

Reply:
great, works perfectly.